terraform/modules/aws_cargohold-asset-storage/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "region" {
```

Comment:
this variable is never used

Reply:
it is indeed used, but only or generating the output `s3_endpoint`. Though, that specific output is a bit useless IMHO, since it is fairly static.